### PR TITLE
Add close modal button

### PR DESCRIPTION
**Pull request by AmberWilkie**
PT Story: https://www.pivotaltracker.com/story/show/132223709

Changes proposed in this pull request:
- Add X (cancel) button to restaurant show page
- Attach `closeModal()` functionality to button

What I have learned working on this feature: 
- How to properly reference ion icons
- How to set opacity on CSS elements
- How to position CSS elements `fixed` on IOS and Android (they are different!)

Screenshots:
screen shot 2016-10-13 at 11 24 58 am

Ready for review!

### CODE REVIEW

File: www/templates/show-restaurant.html [hunk 1]
```diff
@@ -13,5 +13,10 @@ <h4 class'rest-desc'>{{restaurant.description}}</h4>
             {{menu.title}}
           </a>
         </div>
+        <div class="close-button">
+            <a href='#' ng-click='closeModal()'>
```
Comment (tochman) [inline www/templates/show-restaurant.html]:
```
<div class="close-button" ng-click='closeModal()' >
```

Comment (AmberWilkie) [reply www/templates/show-restaurant.html]: done

File: www/templates/show-restaurant.html [hunk 2]
```diff
@@ -14,8 +14,7 @@ <h4 class'rest-desc'>{{restaurant.description}}</h4>
           </a>
         </div>
     </ion-content>
-    <div class="close-button">
-        <a href='#' ng-click='closeModal()'>
+    <div class="close-button" ng-click="closeModal()">
             <i class='icon cancel-icon ion-android-cancel'></i>
         </a>
```
Comment (tochman) [inline www/templates/show-restaurant.html]: But you left the closing tag in place...

Comment (AmberWilkie) [reply www/templates/show-restaurant.html]: ok it's gone now

**Pull request merged**